Doc/library/unicodedata.rst

```diff
@@ -234,7 +274,7 @@ In addition, the module exposes the following constant:
 
 .. data:: ucd_3_2_0
 
-   This is an object that has the same methods as the entire module, but uses the
+   This is an object that has most of the methods of the entire module, but uses the
```

Comment:
This sentence is not fully right, but I can’t find the right suggestion with both «most of» and «same as».